packages/api/test/__fixtures__/sdk/optional-payload/schemas.ts

```diff
@@ -1,4 +1,4 @@
-const updatePetWithForm = {
+const UpdatePetWithForm = {
```

Comment:
With the work to ensure that variables and methods I'm creating aren't reserved I'm capitalizing these schema names here. They're only used within the `types.ts` for passing into [json-schema-to-ts](https://npm.im/json-schema-to-ts) so their name doesn't matter _that_ much, just as long as it doesn't break anything I'm happy.